packages/api-gateway/src/drivers/http/adapters/index.js

```diff
@@ -6,6 +6,7 @@ const adminPanelAdapters = require('./admin_panel/handlers');
 const notificationsAdapters = require('./notifications/handlers');
 const placesAdapters = require('./places/handlers');
 const messageAdapters = require('./messages/handlers');
+const userAdapter = require("./users/handlers");
```

Comment:
ESLint is throwing errors because this line use double-quotes instead of single-quotes, Can you change it?
```suggestion
const userAdapter = require('./users/handlers');
```